test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -711,8 +711,62 @@ namespace StackTest {
             $longPipeName = [string]::new("A", 200)
 
             "`$pid" | & $powershell -CustomPipeName $longPipeName -c -
-            # 64 is the ExitCode for BadCommandLineParameter
-            $LASTEXITCODE | Should -Be 64
+            $LASTEXITCODE | Should -Be $ExitCodeBadCommandLineParameter
+        }
+    }
+
+    Context "ApartmentState WPF tests" -Tag Slow {
+
+        It "WPF requires STA and will work" -Skip:(!$IsWindows) {
+            add-type -AssemblyName presentationframework
+
+            $xaml = [xml]@"
+            <Window
+                xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                x:Name="Window" Title="Initial Window" WindowStartupLocation = "CenterScreen"
+                Width = "400" Height = "300" ShowInTaskbar = "True">
+            </Window>
+"@
+
+            $reader = [System.Xml.XmlNodeReader]::new($xaml)
+            $Window = [System.Windows.Markup.XamlReader]::Load($reader)
+            # This will throw an exception if MTA
+            { $Window.Show() } | Should -Not -Throw
+            $Window.Close()
+        }
+
+    }
+
+    Context "ApartmentState tests" {
+
+        It "Default apartment state for main thread is STA" -Skip:(!$IsWindows) {
+            [System.Threading.Thread]::CurrentThread.ApartmentState | Should -BeExactly "STA"
```

Comment:
```suggestion
            [System.Threading.Thread]::CurrentThread.GetApartmentState()| Should -BeExactly "STA"
```

Reply:
Will fix.  Your suggestion is missing whitespace before the pipe, so will manually fix.